jekyll/_config.yml

```
@@ -19,6 +19,11 @@ algolia:
   application_id: U0RXNGRK45
   api_key: dd49f5b81d238d474b49645a4daed322 # search-only API Key; safe for front-end code
   index_name: documentation
+  extensions_to_index:
+    - html
+    - md
+    - adoc
+  nodes_to_index: 'p,li,td,pre,h1,h1,h3'
```

```suggestion
  nodes_to_index: 'p,li,td,pre,h1,h2,h3'
```

I'm guessing this should be `h2`?

---

Oh yeah 🤦